programs/manifest/src/state/market.rs

```diff
@@ -861,10 +861,8 @@ impl<Fixed: DerefOrBorrowMut<MarketFixed>, Dynamic: DerefOrBorrowMut<[u8]>>
             }
         }
 
-        // TODO: Remove the silent failures
-        // Silently fails because order could have matched before our cancel got
```

Comment:
Is this race condition no longer possible?

Reply:
Wrapper handles it